test/kmc/ut_mariadb.c

```diff
@@ -40,10 +40,12 @@
 
 void cleanup_sa(SecurityAssociation_t* test_association)
 {
-    if (test_association->ek_ref != NULL)
-        free(test_association->ek_ref);
-    if (test_association->ak_ref != NULL)
-        free(test_association->ak_ref);
+    if (test_association->ek_ref[0] != '\0')
+        clean_ek_ref(test_association);
+        // free(test_association->ek_ref);
```

Comment:
Same a mallocs above, ought to just remove instead of leave commented out in code